Source/WebCore/platform/audio/PlatformMediaSessionManager.cpp

```diff
@@ -91,6 +91,13 @@ void PlatformMediaSessionManager::updateNowPlayingInfoIfNecessary()
         existingManager->scheduleSessionStatusUpdate();
 }
 
+void PlatformMediaSessionManager::updateAudioSessionCategoryIfNecessary()
+{
+    fprintf(stderr, "PlatformMediaSessionManager::updateAudioSessionCategoryIfNecessary %p\n", PlatformMediaSessionManager::sharedManagerIfExists());
```

Comment:
Oops, don't land with this!

Reply:
Gasp, I did not see your comment before merging... will fix it shortly

Reply:
https://github.com/WebKit/WebKit/pull/7319